fecgov/fec-cms
# [HOTFIX - MERGE WITH GITFLOW] 5977 Map Tiles Provider(s)

## Summary

- Resolves #5977

Trying to solve the issue of Stamen maps being moved to Stadia. Everything is in place on Feature and seems to be working as expected—functionality and Stadia's dashboard is showing usage.


### Required reviewers

Two? Feel free to add/remove

## Impacted areas of the application

**Sitewide**
This sets `SECURE_REFERRER_POLICY` to `"strict-origin-when-cross-origin"` so external sites can see our domain name when we make requests from them (instead of `null`), necessary for validating with stadiamaps.com

**Maps**
Election search maps at
- [/](http://127.0.0.1:8000/)
- [/data/](http://127.0.0.1:8000/data/)
- [/data/elections/](http://127.0.0.1:8000/data/elections/)

## Screenshots

BROKEN
<img width="482" alt="image" src="https://github.com/fecgov/fec-cms/assets/26720877/6d6fcb46-2d3b-483a-ac8b-cc55768189c6">

<img width="475" alt="image" src="https://github.com/fecgov/fec-cms/assets/26720877/8b41137a-a6e0-46a3-8e37-36a78ed61423">


FIXED
<img width="477" alt="image" src="https://github.com/fecgov/fec-cms/assets/26720877/91c1c6a6-7250-4d95-8e33-e663cde021f5">

<img width="475" alt="image" src="https://github.com/fecgov/fec-cms/assets/26720877/de7e1ce9-1133-499b-8b88-157a985ebb71">


## Related PRs

Not really

## How to test

1) Check
   a. Feature, or
   b. localhost (pull the branch locally, `npm i`, and `npm run build`)
2) play with the map, dragging and zooming it
3) let me know if you see
   - any of the error messages and
   - if a reload fixes it and
   - if another shift-reload breaks it again
